.gitignore

```diff
@@ -0,0 +1,29 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+env/
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+*.egg-info/
+.installed.cfg
+*.egg
+*.pyc
+
+.idea/
+.vscode/
+venv
```

Comment:
Is this really necessary?

Reply:
yeah, this is very important so that we dont add unnecessary files to the repository, such as precompiled py code and virtual env, that weighs heavily

![image](https://user-images.githubusercontent.com/60560085/146488046-583646a1-c3d3-43d3-9aac-98e07c285648.png)

for the compiled code it doest weigh so much, after cleaning it doest remove almost anything as we can see, but its good not to have them in the repository

![image](https://user-images.githubusercontent.com/60560085/146488458-6bea8256-8864-4a75-9efc-00249a2a090b.png)

![image](https://user-images.githubusercontent.com/60560085/146488446-112fbde6-074c-401d-a444-babee08e79fd.png)

Reply:
Going to merge it! Thanks.